### PR TITLE
[v6r16] Added environment variables to rc files to enable certificates verification

### PR DESCRIPTION
**Pull request by fstagni**

### CODE REVIEW

File: Core/scripts/dirac-install.py [hunk 1]
```diff
@@ -1352,6 +1352,9 @@ def createBashrc():
                      '( echo $PYTHONPATH | grep -q $DIRAC ) || export PYTHONPATH=$DIRAC:$PYTHONPATH'] )
       lines.extend( ['# new OpenSSL version require OPENSSL_CONF to point to some accessible location',
                      'export OPENSSL_CONF=/tmp'] )
+      lines.extend( ['# CAs path for SSL verification',
```
Comment (andresailer) [inline Core/scripts/dirac-install.py]: There are checks above (line 1334ff) if /etc/grid-security/certificates is installed locally, this should be taken into account here as well (same for the cshrc)

Comment (fstagni) [inline Core/scripts/dirac-install.py]: Correct. Sorry, I didn't see you message before merging, I will make a further change. Thanks

Comment (fstagni) [inline Core/scripts/dirac-install.py]: I am a bit confused here (cc-ing also @closier after today's comment, and @atsareg )

First of all, taking a random LHCb server:

```
[dirac@lbvobox108 pro]$ echo $X509_CERT_DIR 
/etc/grid-security/certificates
```
which is ~expected (?) but then:

```
[dirac@lbvobox108 dirac]$ more bashrc 
# DIRAC bashrc file, used by service and agent run scripts to set environment
export PYTHONUNBUFFERED=yes
export PYTHONOPTIMIZE=x
[ -z "$HOME" ] && export HOME=/home/dirac
export X509_VOMS_DIR=/opt/dirac/pro/etc/grid-security/vomsdir
# Some DIRAC locations
export DIRAC=/opt/dirac/pro
export DIRACBIN=/opt/dirac/pro/Linux_x86_64_glibc-2.12/bin
export DIRACSCRIPTS=/opt/dirac/pro/scripts
export DIRACLIB=/opt/dirac/pro/Linux_x86_64_glibc-2.12/lib
export TERMINFO=/opt/dirac/pro/Linux_x86_64_glibc-2.12/share/terminfo:/usr/share/terminfo:/etc/terminfo
export RRD_DEFAULT_FONT=/opt/dirac/pro/Linux_x86_64_glibc-2.12/share/rrdtool/fonts/DejaVuSansMono-Roman.ttf
# Prepend the PYTHONPATH, the LD_LIBRARY_PATH, and the DYLD_LIBRARY_PATH
( echo $PATH | grep -q $DIRACBIN ) || export PATH=$DIRACBIN:$PATH
( echo $PATH | grep -q $DIRACSCRIPTS ) || export PATH=$DIRACSCRIPTS:$PATH
( echo $LD_LIBRARY_PATH | grep -q $DIRACLIB ) || export LD_LIBRARY_PATH=$DIRACLIB:$LD_LIBRARY_PATH
( echo $LD_LIBRARY_PATH | grep -q $DIRACLIB/mysql ) || export LD_LIBRARY_PATH=$DIRACLIB/mysql:$LD_LIBRARY_PATH
( echo $DYLD_LIBRARY_PATH | grep -q $DIRACLIB ) || export DYLD_LIBRARY_PATH=$DIRACLIB:$DYLD_LIBRARY_PATH
( echo $DYLD_LIBRARY_PATH | grep -q $DIRACLIB/mysql ) || export DYLD_LIBRARY_PATH=$DIRACLIB/mysql:$DYLD_LIBRARY_PATH
( echo $PYTHONPATH | grep -q $DIRAC ) || export PYTHONPATH=$DIRAC:$PYTHONPATH
# new OpenSSL version require OPENSSL_CONF to point to some accessible location
export OPENSSL_CONF=/tmp
# GFAL2
export GFAL_CONFIG_DIR=/opt/dirac/pro/Linux_x86_64_glibc-2.12/etc/gfal2.d
export GFAL_PLUGIN_DIR=/opt/dirac/pro/Linux_x86_64_glibc-2.12/lib/gfal2-plugins/
# IPv6 support
export GLOBUS_IO_IPV6=TRUE
export GLOBUS_FTP_CLIENT_IPV6=TRUE
# ARC Computing Element
export ARC_PLUGIN_PATH=$DIRACLIB/arc
```

so no mention of `X509_CERT_DIR`, while looking at the code I expect to see it: https://github.com/DIRACGrid/DIRAC/blob/integration/Core/scripts/dirac-install.py#L1331 
Am I misunderstanding something here? (And, BTW, looking at another LHCb vo-box, `X509_CERT_DIR` is not set at all... which of course should not be a real problem).

Second, in that same code linked above it is "expected" that the only place where CAs may be found is in /etc/grid-security/certificates which in fact is not necessarily the case (e.g. think at CVMFS directories - where BTW LHCb one is richer than DIRAC one).

Comment (andresailer) [inline Core/scripts/dirac-install.py]: I guess whoever (whatever?) ran dirac-install didn't have X509_CERT_DIR set?

**Pull request merged**